tests/scripts/pulp_container/test_distribution.sh

```diff
@@ -25,10 +25,12 @@ expect_succ pulp container distribution create --name "cli_test_container_distro
 
 expect_succ pulp container distribution list
 expect_succ pulp container distribution show --name "cli_test_container_distro"
-# Not sure what's going on here. Maybe a bug in the distribution serializer.
-# expect_succ pulp container distribution update --name "cli_test_container_distro" --version 0
-# expect_succ pulp container distribution update --name "cli_test_container_distro" --repository ""
-# expect_succ pulp container distribution update --name "cli_test_container_distro_ver" --repository "cli_test_container_repository"
-# expect_succ pulp container distribution update --name "cli_test_container_distro_ver" --repository ""
 
+if pulp debug has-plugin --name "container" --min-version "2.10.0.dev"
```

Comment:
So we accept that this is a bug we could not work around in earlier versions? :+1:
Optional: Leave a link to the issue in `pulp_container` here.

Reply:
yeah, unfortunately. There was a fix in core *and* plugin, so this combination basically makes it impossible to make the tests pass against earlier versions :/